gitlab-runner/src/run.rs

```diff
@@ -247,7 +247,12 @@ impl Run {
         }
     }
 
-    #[tracing::instrument(skip(self, process,build_dir),fields(gitlab.job=self.response.id))]
+    // the gitlab_runner::gitlab::job target is specified for crate users to depend on; Also it's
+    // at max verbosity (error), so it always shows up regardless of level.
+    #[tracing::instrument(target="gitlab_runner::gitlab::job",
+                          level = "error",
```

Comment:
nit: mix of `=` with spaces on the this line and without spaces on the line above & below in the `fields`.